### ISSUE TITLE
[A11y] Tooltip is not accessible with keyboard for tree items

### ISSUE THREAD

**Issue by HuihuiWu-Microsoft** [microsoft/vscode#277997]
Does this issue occur when all extensions are disabled?: Yes/No

- VS Code Version: Version: 1.105.1 (user setup)
Commit: 7d842fb85a0275a4a8e4d7e040d2625abbf7f084
Date: 2025-10-14T22:33:36.618Z
Electron: 37.6.0
ElectronBuildId: 12502201
Chromium: 138.0.7204.251
Node.js: 22.19.0
V8: 13.8.258.32-electron.0
OS: Windows_NT x64 10.0.26200
- OS Version: 

Steps to Reproduce:

1. Use keyboard to switch between tree items
2. Noticed the tooltip is not present
